docker-compose-postgres.yaml

```diff
@@ -57,15 +57,16 @@ services:
   golem-template-service:
     build:
       context: .
-      dockerfile: golem-service/docker/Dockerfile
+      dockerfile: golem-template-service/docker/Dockerfile
+    restart: on-failure
     environment:
       - ENVIRONMENT=local
       - WASMTIME_BACKTRACE_DETAILS=1
       - RUST_BACKTRACE=1
       - RUST_LOG=info,h2=warn,hyper=warn,tower=warn
       - GOLEM__TEMPLATE_STORE__TYPE="Local"
       - GOLEM__TEMPLATE_STORE__CONFIG__OBJECT_PREFIX=""
-      - GOLEM__TEMPLATES_STORE__CONFIG__ROOT_PATH=/template_store
+      - GOLEM__TEMPLATE_STORE__CONFIG__ROOT_PATH=/template_store
```

Comment:
I think I fixed it already

Reply:
this PR is mainly about postgres schema configuration, i had to fix this docker compose, to test it, as your PR was not merged yet at that time